scripts/src/build.sh

```diff
@@ -14,7 +14,9 @@ build_usage() {
     help           show help message and exit
 
     gear           build gear workspace
-    examples       build gear program examples
+    examples       build gear program examples,
+                   you can specify yaml list to build coresponding examples
+                   using yamls="path/to/yaml1 path/to/yaml2 ..." argument
```

Comment:
I've heard somewhere that 'explicit is better than implicit' so I'd prefer to have possibility to explicitly set which examples should be built. However if the very same feature to build all examples mentioned in yaml is required then maybe it is better to introduce new target?

Another concern is how to build all examples with this approach?

Reply:
> Another concern is how to build all examples with this approach?
You can skip yamls argument in that case all examples will be built.
This argument is created to build all examples deps in yamls, so if you want to build special example - you can make your own argument))) or make your own yaml file.
Also, I create this arg to be able run specified test  and avoid building all examples each time.